torch/csrc/jit/runtime/interpreter.h

```diff
@@ -98,8 +99,11 @@ struct InterpreterContinuation {
   InterpreterContinuation(
       InterpreterState state_,
       Stack stack_,
-      int64_t dist_autograd_context_id = 0)
-      : state(state_), stack(std::move(stack_)) {
+      int64_t dist_autograd_context_id = 0,
+      c10::optional<at::ThreadLocalState> tls_state = c10::nullopt)
```

Comment:
just curious - can you remind me the reason dist_autograd_context is not part of TLS? (I think there was one but I don't recall)

Reply:
I better refer to https://github.com/pytorch/pytorch/issues/38510 , there's been some ongoing discussion